config/gnu-cxxflags

```diff
@@ -239,30 +240,45 @@ if test "X-g++" = "X-$cxx_vendor"; then
 
     # gcc >= 6
     if test $cxx_vers_major -ge 6; then
+        # Use the C warnings as CXX warnings are the same
         H5_CXXFLAGS="$H5_CXXFLAGS $(load_gnu_arguments 6)"
     fi
 
     # gcc >= 7
     if test $cxx_vers_major -ge 7; then
+        # Use the C warnings as CXX warnings are the same
         H5_CXXFLAGS="$H5_CXXFLAGS $(load_gnu_arguments 7)"
         DEVELOPER_WARNING_CXXFLAGS="$DEVELOPER_WARNING_CXXFLAGS $(load_gnu_arguments developer-7)"
     fi
 
     # gcc 8
     if test $cxx_vers_major -ge 8; then
+        # Use the C warnings as CXX warnings are the same
         H5_CXXFLAGS="$H5_CXXFLAGS $(load_gnu_arguments 8)"
         #H5_ECXXFLAGS="$H5_ECXXFLAGS $(load_gnu_arguments error-8)"
         #H5_NECXXFLAGS="$H5_NECXXFLAGS $(load_gnu_arguments noerror-8)"
+        # Use the C warnings as CXX warnings are the same
         DEVELOPER_WARNING_CXXFLAGS="$DEVELOPER_WARNING_CXXFLAGS $(load_gnu_arguments developer-8)"
         NO_DEVELOPER_WARNING_CXXFLAGS="$NO_DEVELOPER_WARNING_CXXFLAGS $(load_gnu_arguments no-developer-8)"
     fi
 
     # gcc 9
     if test $cxx_vers_major -ge 9; then
-        H5_CXXFLAGS="$H5_CXXFLAGS $(load_gnu_arguments 9)"
         H5_CXXFLAGS="$H5_CXXFLAGS $(load_gnu_arguments cxx-9)"
     fi
 
+    # gcc >= 9.3
+    if test $cc_vers_major -ge 10 -o $cc_vers_major -eq 9 -a $cc_vers_minor -ge 3; then
+        # do not use C warnings, $(load_gnu_arguments 9.3), no cxx warnings
+        # H5_CXXFLAGS="$H5_CXXFLAGS $(load_gnu_arguments cxx-9.3)"
```

Comment:
I think we can delete the commented-out load_gnu_arguments lines carried over from gnu-flags, etc.